README.md

```diff
@@ -1,2 +1,55 @@
 # template-talon_standard
-A robot program template for easy programming of a swerve drive and arm powered by KrakenX60 motors.
+
+This template standard contains code for two main types of sensors, photo-electric, and gyroscopes.
+
+This code template will explain how to use beam break and color sensors.
+
+The first type of sensor, photo-electric:
+
+Photo-electric sensors work by emitting a laser to a surface. Once an element emits the light, the other element tries to receive it. If the component gets the light, that means that nothing is obstructing the light, in other words, no object. If there is an object, the object will block the light, and the receiver will never get the light, meaning that there is an object.
+
+Example Diagram:
+![image](https://github.com/user-attachments/assets/e8e93da7-e164-4a0c-b597-9b4215b70131)
+
```

Comment:
Doesn't that diagram show a sensor that detects reflected light, rather than a beam-break as described above?

Reply:
Its called a photo electric sensor formally. if I said beam-break i will change it. Beam break is different because it has two different parts (laser emitter, and respondent). The photo electric sensor is only one part.

Reply:
"Once an element emits the light, the other element tries to receive it. If the component gets the light, that means that nothing is obstructing the light, in other words, no object."

That describes beam break, not reflective.

Reply:
Docs need clarification.